x-pack/plugins/ml/server/routes/saved_objects.ts

```diff
@@ -251,31 +251,31 @@ export function savedObjectsRoutes(
   /**
    * @apiGroup JobSavedObjects
    *
-   * @api {post} /api/ml/saved_objects/remove_job_from_current_space Remove jobs from the current space
+   * @api {post} /api/ml/saved_objects/remove_item_from_current_space Remove jobs from the current space
    * @apiName RemoveJobsFromCurrentSpace
    * @apiDescription Remove a list of jobs from the current space.
```

Comment:
Please update the API description

Reply:
https://github.com/elastic/kibana/pull/127438/commits/b7fffe7d26974f5918ba0de21dd87318afb05ff0